app/views/site/id.html.erb

```diff
@@ -15,7 +15,15 @@
      data[:token] = token.token
    end
    data[:locale] = ID::LOCALES.preferred(preferred_languages).to_s
-   data[:asset_map] = assets("iD").to_json %>
+   data[:asset_map] = assets("iD").to_json
+
+  # Include user preferences for iD editor
+   if current_user
```

Comment:
it's probably not required to check for `current_user` as this page requires to be logged in to load, and the variable is already used on line 14.